javascript-plugin/src/main/java/fr/greencodeinitiative/javascript/JavaScriptRulesDefinition.java

```diff
@@ -6,8 +6,8 @@
 public class JavaScriptRulesDefinition implements RulesDefinition {
 
     private static final String LANGUAGE = "js";
-    private static final String LINTER_KEY = "eslint_repo";
-    private static final String LINTER_NAME = "ESLint";
+    private static final String LINTER_KEY = "ecocode-js";
+    private static final String LINTER_NAME = "ecoCode";
```

Comment:
As we integrate our rules [using SonarJS](https://github.com/SonarSource/SonarJS/blob/b15f4cede8e6a506d26b3fa99cf1923b5cca21a8/sonar-plugin/sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/rules/EslintRulesDefinition.java#L30-L31), these values should remain to "eslint_repo" and "ESLint" in order to import them correctly. I have done few tests with proposed values and it does not work: rules appear as "removed" and do not have tags

Reply:
sorry, I will revert those changes.